test/system/application-scans/python.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { scan } from "../../../lib";
+import { getFixture } from "../../util";
+
+describe("python application scans", () => {
```

Comment:
What do you think about splitting the tests for `poetry`, `pip` and `pipenv` to different files? They don't actually have anything in common, and that would enable better run times for the tests.

Reply:
Sure. Once we'll have more package managers we support, we can split the tests to different files

Reply:
I actually think it's better to do it now if it is your intention. It would make the next PR easier to review, and generally allows the reviewer to better understand the test file's purpose.